modules/apps/object/object-api/src/main/resources/com/liferay/object/service/packageinfo

```diff
@@ -1 +1 @@
-version 65.1.0
\ No newline at end of file
+version 66.0.0
```

Comment:
Are you sure this caused a breaking change? You only changed some logic in a private method.
If you really caused a breaking change, please move the message to the commit who caused it.